### DISCUSSION TITLE
Choose the appropriate mui icon

### DISCUSSION THREAD

**Discussion by Kim-Jiyun** [AgainIoT/Open-Set-Go#55] [Feature Suggestion / Share Ideas]
I've found mui icons as several candidates and I'm posting because I want to pick the right one among them! Take a look all and let us know which icon you think is more suitable!
mui 아이콘을 여러 후보로 찾았는데 그 중 알맞는 아이콘을 고르고 싶어서 글을 올립니다! 모두 보시고 어떤 아이콘이 더 적합하다고 생각하는지 알려주세요!

1. Code of Conduct(행동 강령)
![rule](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/11d115ef-9234-4477-9d5e-b605e7931c46)
![gavel](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/72cf55fa-c64d-4e7d-9cc1-e3e1147924aa)

2. License
![announcement](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/ee49bd98-df56-4616-bf91-b3e14029e292)
![privacytip](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/881798bd-adcc-442a-b51b-c496083e7e1c)

3. Secret Scanning
![lock](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/7a623562-20a4-45db-827d-ee9677572b3d)
![lockreset](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/7d43f8f1-681b-4e4c-93e8-51f45734368b)

4. Security Policy
![security](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/ca09ba80-d9e3-461f-9b72-2481a3fc7b8f)
![policy](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/6bcb7dca-6215-4487-8941-2db8dd405e7e)

I thought it would be okay, so I looked more icons.
괜찮을 것 같아서 더 찾아본 아이콘들 입니다.
![sdcardalert](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/de38bf36-432b-440e-91d6-4fca4ca26a5c)
![securityupdategood](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/f231c280-c775-4263-9421-9c85c00d6d1a)

**Comment by ymw0407** [comment]
My Vote
---
Code of Conduct - Gavel
Secret Scanning - Policy
Security Policy - Security

I think we should use the logos here.
여기 있는 로고들은 그대로 사용해야할 것 같습니다.
![image](https://github.com/AgainIoT/Open-Set-Go/assets/77202633/61c88f4c-b19e-4b4c-a695-bd07ce24c8c6)

**Reply by ancy0** [reply]
As you said, the item attached to the picture will use the icon on the main page as it is.
사진에 첨부된 항목에 대해서는 메인페이지의 아이콘을 그대도 사용할 예정입니다.

**Comment by ancy0** [comment]
To add a description of what the icon needs, it's as below.
아이콘이 필요한 항목에 대한 설명을 덧붙이자면 아래와 같습니다.

**1. Code of Conduct**
A code of conduct is a document that establishes expectations for behavior for your project’s participants. (Adopting, and enforcing, a code of conduct can help create a positive social atmosphere for your community.) 
**3. Secret Scanning**
Get notified when a secret is pushed to this repository
**4. Security Policy**
View how to securely report security vulnerabilities for this repository

😄 License will use an existing icon.

**Comment by hummingbbird** [comment]
1. Code of Conduct: Gavel
2. License: PrivacyTip
3. Secret Scanning: LockReset
4. Security policy: Security

**Comment by ancy0** [comment]
My Vote
---

[1] Code of Conduct -> _Gavel_
[4] Security Policy -> _Security_

[2] License -> we will use an existing icon.
[3] Secret Scanning -> plz suggest new icon :)

**The meaning of each item should be clearly revealed.**

**Reply by ancy0** [reply]
For your information, License is using the Lock icon shown above.

**Comment by sinji2102** [comment]
1. Code of Conduct: Rule
2. License: PrivacyTip
3. Secret Scanning: Lock
4. Security policy: Security

**Comment by Kim-Jiyun** [comment]
To sum up the opinions, I think we can use
Code of Conduct is a Gavel icon
License is the icon that you originally used (Lock icon)
Security Policy is a Security icon!
의견을 총 정리하면
Code of Conduct는 Gavel 아이콘으로
License는 원래 사용하던 아이콘(lock 아이콘)으로
Security policy은 Security 아이콘으로 하면 될 것 같습니다!

No. 3 Secret Scanning played the role of 'Get notified when a secret is pushed to this repository', so we decided that Notifications Activeicon was more appropriate!
3번  Secret Scanning은 'Get notified when a secret is pushed to this repository' 를 하는 역할로 NotificationsActive icon이 더 적절하다고 판단했습니다!
![notificationsactive](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/2cd836a3-de1c-41d4-aff9-2c9ae273f530)


Thank you to everyone who commented.
의견 주신 모든 분께 감사합니다.
